code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -195,6 +195,15 @@
 			new /obj/item/dnainjector/cryokinesis(src)
 			new /obj/item/gun/energy/temperature/security(src)
 			new /obj/item/melee/transforming/energy/sword/saber/blue(src) //see see it fits the theme bc its blue and ice is blue
+			
+		if("ancient") //A kit so old, it's probably older than you.
+			new /obj/item/card/emag(src)
+			new /obj/item/pen/sleepy(src)
+			new /obj/item/reagent_containers/pill/cyanide
+			new /obj/item/chameleon(src)
+			new /obj/item/gun/ballistic/revolver(src)
+			new /obj/item/implanter/freedom(src)
+			new /obj/item/stack/telecrystal(src) //The failsafe/self destruct isn't an item we can physically include in the kit, but 1 TC is technically enough to buy the equivalent.
```

Comment:
maybe this kit could come with a piece of paper that explains what this telecrystal is intended to be used for?

Reply:
I am fairly sure that it's possible to have the self destruct included in the kit on purchase, but it'll be a little fiddly.

Reply:
Yeah, but ik personally not cool enough to know how to do this.